packages/augur-test/tests/state/api/Markets.test.ts

```diff
@@ -0,0 +1,193 @@
+import { API } from "@augurproject/state/src/api/API";
+import { MarketInfo } from "@augurproject/state/src/api/Markets";
+import { MarketType } from "@augurproject/state/src/api/types";
+import { DB } from "@augurproject/state/src/db/DB";
+import {
+  ACCOUNTS,
+  makeDbMock,
+  compileAndDeployToGanache,
+  ContractAPI,
+} from "../../../libs";
+import { stringTo32ByteHex } from "../../../libs/Utils";
+
+const mock = makeDbMock();
+
+beforeEach(async () => {
+  mock.cancelFail();
+});
+
+let db: DB<any>;
+let api: API<any>;
+let john: ContractAPI;
+let mary: ContractAPI;
```

Comment:
You can delete mary since you don't use her.

Reply:
I'm planning to use mary eventually for trading to test volume/OI, so that's why I left her in this test.